blivet/mounts.py

```diff
@@ -32,29 +32,42 @@ class MountsCache(object):
 
     def __init__(self):
         self.mountsHash = 0
-        self.mountpoints = {}
+        self.mountpoints = defaultdict(list)
 
-    def getMountpoint(self, devspec, subvolspec=None):
-        """ Get mountpoint for selected device
+    def getMountpoints(self, devspec, subvolspec=None):
+        """ Get mountpoints for selected device
 
             :param devscpec: device specification, eg. "/dev/vda1"
             :type devspec: str
             :param subvolspec: btrfs subvolume specification, eg. ID or name
             :type subvolspec: str
-            :returns: mountpoint (path)
-            :rtype: str or NoneType
+            :returns: list of mountpoints (path)
+            :rtype: list of str or empty list
```

Comment:
Probably useful to include a note about the order that the mountpoints occur in the result list.